packages/widgets/src/lib/components/InferenceWidget/widgets/AudioClassificationWidget/AudioClassificationWidget.svelte

```diff
@@ -156,43 +156,40 @@
 </script>
 
 <WidgetWrapper
-	{callApiOnMount}
 	{apiUrl}
 	{includeCredentials}
-	{applyInputSample}
-	{computeTime}
-	{error}
-	{isLoading}
 	{model}
-	{modelLoading}
-	{noTitle}
-	{outputJson}
-	{validateExample}
+	let:isDisabled
+	let:modelLoadInfo
+	let:WidgetInfo
+	let:WidgetHeader
+	let:WidgetFooter
```

Comment:
Maybe it would be simpler / cleaner if `isDisabled` / `modelLoadInfo` were in a store?
From what I understand they are only exposed to be "re-injected" in `WidgetHeader` / `WidgetInfo`, correct?

Reply:
handled in 9861a29480af975b2b37953aa9f621df87e9833c